index.html

```diff
@@ -158,7 +159,7 @@ <h1 class="style">My recent work</h1>
     </div>
```

Comment:
- [ ] It is a better practice to have one `h1` tag per page. This helps with SEO. For your page, your name should keep the `h1` tag. My recent work should be `h2` tag and then the project title should have `h3` tag. Ensure to maintain a meaningful hierarchy of heading levels. Make all the changes for your `hx` tags